Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -275,13 +282,108 @@ def __init__(self):
             callback=self._on_target_class_changed,
             contentsLength=8)
 
+        box = gui.vBox(self.controlArea, "Model Comparison")
+        gui.comboBox(
+            box, self, "comparison_criterion", callback=self.score_changed)
+
+        hbox = gui.hBox(box)
+        gui.checkBox(hbox, self, "use_rope",
+                     "Negligible difference: ",
+                     callback=self.score_changed)
+        gui.lineEdit(hbox, self, "rope", validator=QDoubleValidator(),
+                     controlWidth=70, callback=self.score_changed,
+                     alignment=Qt.AlignRight)
+
         gui.rubber(self.controlArea)
         self.score_table = ScoreTable(self)
         self.score_table.shownScoresChanged.connect(self.update_stats_model)
+        view = self.score_table.view
+        view.horizontalHeader().sortIndicatorChanged.connect(self.score_changed)
+        view.setSizeAdjustPolicy(view.AdjustToContents)
 
         box = gui.vBox(self.mainArea, "Evaluation Results")
         box.layout().addWidget(self.score_table.view)
 
+        self.compbox = box = gui.vBox(self.mainArea, box="Model comparison")
+        table = self.comparison_table = QTableWidget(
+            wordWrap=False, editTriggers=QTableWidget.NoEditTriggers,
+            selectionMode=QTableWidget.NoSelection)
+#        table.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+ #       table.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        table.setSizeAdjustPolicy(table.AdjustToContents)
+
+        header = table.horizontalHeader()
+        header.setSectionResizeMode(QHeaderView.ResizeToContents)
+        header.setDefaultAlignment(Qt.AlignLeft)
+        header.setStretchLastSection(False)
+        box.layout().addWidget(table)
+        box.layout().addWidget(QLabel(
+            "<small>Table shows probabilities that the score for the model in "
+            "the row is higher than that of the model in the column. "
+            "Small numbers show the probability that the difference is"
```

Comment:
space after final 'is'

Reply:
That said, that should probably be in the message above, not fixed below the table.

Reply:
I added the space, but would prefer having this text visible.